src/In.ProjectEKA.HipLibrary/Patient/Model/KeyStructure.cs

```diff
@@ -0,0 +1,16 @@
+namespace In.ProjectEKA.HipLibrary.Patient.Model
+{
+    public class KeyStructure
+    {
+        public string Expiry { get; }
```

Comment:
If it's datetime, can you use DateTime only.

Reply:
It is defined as date time in contract

Reply:
Json does not support DateTime :)